cachito/web/models.py

```diff
@@ -77,10 +78,19 @@ class Dependency(db.Model):
         db.UniqueConstraint('name', 'type', 'version'),
     )
 
+    def __repr__(self):
+        return f'<Package id={self.id}, name={self.name} type={self.type} version={self.version}>'
```

Comment:
Can we just use `self.__class__.__name__` so we don't have to override it on every sub class?